src/lib.rs

```diff
@@ -32,7 +32,7 @@ pub struct FormatOptions {
     /// When set, changes reserved keywords to ALL CAPS
     ///
     /// Default: false
-    pub uppercase: bool,
+    uppercase: Option<bool>,
```

Comment:
Did you intend to make this _uppercase_ private?

(thanks for making this fix! I was fixing a case handling problem in another project and noticed that it breaks with this commit. unless I'm using this lib wrong(?))

Reply:
@marcusadair Thank you for pointing out this error. Have Done. :)